package.json

```diff
@@ -31,6 +34,7 @@
     "@nestjs/graphql": "^9.1.2",
     "@nestjs/platform-express": "^8.2.5",
     "@nestjs/typeorm": "^8.0.3",
+    "@unique-nft/sdk": "^0.3.16",
```

Comment:
Shouldn't this package be added to devDependencies instead?

Reply:
moved